data/tables/ali2020_tables_table_4.csv

```diff
@@ -0,0 +1,9 @@
+t (h),W_(L)(mg),,,,
+%NaCl,0,2,3,4,5
+240,0.216,0.248,0.293,0.318,0.343
+480,0.32,0.426,0.446,0.449,0.569
+720,0.475,0.563,0.646,0.662,0.766
+960,0.746,0.822,0.847,0.865,0.942
+1200,0.945,1.007,1.079,1.121,1.191
+1440,1.105,1.197,1.248,1.347,1.402
+1750,1.364,1.55,1.617,1.854,2.171
```

Comment:
Please rename all of these files this shouldn't be tables_table one table should be enough

Reply:
done